commons/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="fr.free.nrw.commons"
-    android:versionCode="18"
-    android:versionName="1.3" >
+    android:versionCode="19"
+    android:versionName="1.31" >
```

Comment:
Did you mean 1.3.1?
I tend to prefer 1.4 rather than using another level, as that makes dichotomies easier.
We already have the first number for "big changes", so my opinion is that the second number can be used generously. By the way, the one after 1.9 will be 1.10. Versio 1.31 will come after 1.29 and 1.30 (if we don't decide to switch to 2.0 before).
Wikimedia considered the app as beta that's why they never released a real 1.0 (always 1.0-SNAPSHOT).